hera_sim/io.py

```diff
@@ -4,14 +4,15 @@
 """
 import numpy as np
 from pyuvsim.simsetup import initialize_uvdata_from_keywords
+from .defaults import _defaults
 
 HERA_LAT_LON_ALT = (
     np.degrees(-0.53619179912885),
     np.degrees(0.3739944696510935),
     1073.0000000074506
 )
 
-
+@_defaults
```

Comment:
This is really cool, however I like the idea of making these `model`s. This is more semantic.

Reply:
We could do this with classes also, but I kind of like the deco syntax -- and each model really does only have one function call.